.github/workflows/nightly-elixir.yml

```diff
@@ -0,0 +1,78 @@
+name: Push generated aws-elixir code
+on:
+  push:
+    branches:
+      - master
+  schedule:
+    - cron: '23 0 * * *' ## Scheduled nightly at 00:23
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    container:
+      image: hexpm/elixir:${{ matrix.elixir }}-erlang-${{ matrix.erlang }}-ubuntu-${{ matrix.ubuntu }}
```

Comment:
I think it's better to use the Erlang Foundations' action, since they are the recommended way for Elixir and Erlang projects now. See: https://github.com/erlef/setup-beam#example-erlangotp--elixir-on-ubuntu